test/extended/testdata/cmd/lib/build/environment.sh

```diff
@@ -0,0 +1,281 @@
+#!/usr/bin/env bash
```

Comment:
why have these been copied?

Reply:
to keep the `test-cmd.sh` working, perhaps not all the files here are necessary